app/views/content/is-teaching-right-for-me/behaviour-management.md

```diff
@@ -0,0 +1,95 @@
+
+I always worked with parents and carers as well, sending out postcards or making phone calls to tell them their child's been brilliant that day. It only takes a few seconds to do but the rewards can be huge. I might start a class and tell them I've got two brilliant phone calls home to give out. So they're competing with themselves to try and be the best that they can. But never promise something you're not going to deliver. 
+
+A lot of the worst behaviours are when children feel rejected so they strive towards that. It's about resetting that mindset in that child. So rather than wanting for example to be the class clown, the label they've given themself is reframed so they’re the better learner, or the most organised, or best at sport. It’s recognising where can they achieve. Finding that is the key to getting it right. 
+
+## What support is there for trainee teachers?
+As a teacher you’re never going to be perfect. You’re constantly learning and the variety and support you get through your day through learning all of these different things is really important.
+
+On most teacher training courses you might watch lots of different classes and learn how a particular teacher works with them. Or you might go round with one student and see what a day in their lives looks like, with the different teachers they may have that day. This allows you to pick up strategies other people may use while also seeing what kind of challenges an individual class or child might have during that period of time.
+
+Then you might work with either small groups or one-to-one with pupils. Again you're observing and analysing what's working. You're not alone. You've got a teacher there so if you think something didn't quite land you've got somebody to bounce ideas off and look at how you might change it.
+
+## Is there any support once I become a teacher?
+All schools have behaviour policies in place. So you work within a remit to ensure best practice and that there's consistency with how behaviour is managed as a school. As a teacher you're not expected to manage behaviour on your own, you get an induction on your school's behaviour policies and preferred ways of management.
+
+If your course leads to QTS, for the first two years of teaching as an [early career teacher (ECT)](/support-for-early-career-teachers) you get ongoing training and development that includes effective behaviour management. Plus you'll have a dedicated mentor to give you support. And you've also got other senior members of staff and colleagues who can help. It's not unheard of for somebody to sit down and say they're struggling to manage the behaviour of child A, are there any strategies you've found successful? You're constantly working together to swap ideas, sanctions and bits of information about that child.
```

Comment:
This is the first mention of QTS on this page so it needs explaining.

It also feels a little clunky - could we say something like:

`If you have qualified teacher status (QTS), for the first two years.....`

It's not about the course leading to QTS, it's about whether they **have** QTS (ie have been awarded it at the end of the course)